content/docs/event_loop.mdz

```diff
@@ -3,168 +3,217 @@
  :order 21}
 ---
 
-In addition to threads, Janet comes with another powerful concurrency model out of the box - the event loop. The event loop provides
-concurrency within a single thread by allowing cooperating fibers to yield instead of
-blocking forward progress. This is a form of cooperative multi-threading that can be useful in many applications, like
-situations where there are many concurrent, IO-bound tasks. This means tasks that spend most of their execution time waiting for
-network or disk, rather than using CPU time.
+In addition to threads, Janet comes with another powerful concurrency model
+out of the box - the event loop. The event loop provides concurrency within
+a single thread by allowing cooperating fibers to yield instead of blocking
+forward progress. An event loop is a form of cooperative multi-tasking that
+can be useful in many applications, like situations where there are many
+concurrent, IO-bound tasks. So tasks that spend most of their execution time
+waiting for network or disk, rather than using CPU time.
 
-Most event loop functionality in Janet is exposed through the @code`ev/` module, but many other functions outside this module
-will interact with the event loop.
+Most event loop functionality in Janet exposes through the @code`ev/` module,
```

Comment:
This is an awkward result from a poor heuristic that says that the passive voice is always bad.

Reply:
It indeed is, but I think the biggest problem here is that I forget to remove "in"  before Janet. I know it is anthropomorphization I am guilty of. Or would you rather see me return the passive voice here?